bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -0,0 +1,116 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/instance_admin.h"
+#include "bigtable/client/instance_admin_client.h"
+
+namespace btproto = ::google::bigtable::admin::v2;
+
+namespace {
+
+//! [create instance]
+void CreateInstance(bigtable::InstanceAdmin& instance_admin,
+                    std::string const& project_id,
+                    std::string const& instance_id) {
+  // TODO(#418) implement tests and examples for CreateInstance
+}
+//! [create instance]
+
+//! [list instance]
```

Comment:
Please call the snippet `[list instances]` (plural) for consistency.

Reply:
done.